docs/maintainer/knowledge_base.md

```diff
@@ -1958,21 +1963,32 @@ and add the following to `recipe/conda_build_config.yaml`:
 
 ```yaml
 c_stdlib_version:          # [linux]
-  - "2.17"                 # [linux]
+  - "2.28"                 # [linux]
 ```
 
 This covers the headers/library present at build-time, and will also create
 a corresponding run-export on the `__glibc` virtual package.
 
-You may also need to use a newer docker image by setting the following in the `conda-forge.yml`
-of your recipe and rerendering.
+By default, conda-forge will default to old sysroots on new images, meaning that
```

Comment:
```suggestion
By default, conda-forge build infrastructure uses old sysroots on new images, meaning that
```

This is my only suggestion. Otherwise, LGTM.